### ISSUE TITLE
Shared-styles doesn't work as expected in Firefox ESR

### ISSUE THREAD

**Issue by johannest** [vaadin/flow#5049]
Description of the bug: Using shared-styles.html files to customize e.g. vaadin-button styles doesn't work with Firefox ESR releases.

Minimal reproducible example
```
<dom-module id="custom-button-theme" theme-for="vaadin-button">
    <template>
        <style>
            :host {
                border: 0 none;
                background: #8fa2ac;
                color: #ffffff;
            }
        </style>
    </template>
</dom-module>
```

Expected behavior: The custom style would be applied to component also when using ESR release of FF
Actual behavior: The styles doesn't seem to be applied.
Vaadin / Flow version: 12.0.5
Java version: 1.8.0_181
OS version: Win 10
Browser version (if applicable): 60.5.0esr (64-bit)

**Comment by johannest** [comment]
Seems to affect mostly vaadin-button. For example a custom style for vaadin-text-field works also in FF ESR. Also some variables such as --lumo-border-radius works for vaadin-button, but most of the styles doesn't work.

**Comment by jouni** [comment]
Here’s a pure client-side example, which seems to work fine (Firefox 60.5.1esr, macOS 10.14): https://glitch.com/edit/#!/hail-hygienic?path=index.html:13:2

**Comment by TatuLund** [comment]
> Here’s a pure client-side example, which seems to work fine (Firefox 60.5.1esr, macOS 10.14):

So is it possible that this is issue in Firefox 60.5.0esr, which has been fixed in 60.5.1esr?

**Comment by web-padawan** [comment]
Probably related to vaadin/vaadin-themable-mixin#38

**Comment by Klaudeta** [comment]
setting `dom.webcomponents.customelements.enabled` to true fixes the issue for FF ESR

**Comment by Klaudeta** [comment]
The order in which the html import dependencies url are sent from flow to the client seems to be correct, the shared-style.html is the last dependencies url sent to the client within the response

**Comment by Klaudeta** [comment]
Workaround until it get resolved from ticket [#38](vaadin/vaadin-themable-mixin#38)

is in here [#28(comment)](/vaadin/vaadin-themable-mixin/issues/28#issuecomment-445475643)

**Comment by web-padawan** [comment]
I can confirm that's the correct workaround. Furthermore, the instantiation order is not really specific to HTML imports only, and will likely become a must when Flow switches to ES modules.

**Comment by pleku** [comment]
This has been fixed and included in 13.0.1.
